src/utils/dex/egggroup.js

```diff
@@ -0,0 +1,62 @@
+import { PersonalTable } from '../../../__gamedata';
+
+const EGG_GROUPS = [
+  'None',
+  'Monster',
+  'Water 1',
+  'Bug',
+  'Flying',
+  'Field',
+  'Fairy',
+  'Grass',
+  'Human-Like',
+  'Water 3',
+  'Mineral',
+  'Amorphous',
+  'Water 2',
+  'Ditto',
+  'Dragon',
+  'No Eggs',
+];
```

Comment:
array position == egg group id. maybe explicitly set the egg group id and create a Map instead of an Array. but can be left as it is

```
const EGG_GROUPS = {
  0: 'None',
  1: 'Monster',
  2: 'Water 1',
  3: 'Bug',
  4: 'Flying',
  5: 'Field',
  6: 'Fairy',
  7: 'Grass',
  8: 'Human-Like',
  9: 'Water 3',
  10: 'Mineral',
  11: 'Amorphous',
  12: 'Water 2',
  13: 'Ditto',
  14: 'Dragon',
  15: 'No Eggs',
};
```

Reply:
added those